documentation/OtherMLPackages.md

```diff
@@ -0,0 +1,62 @@
+# Testing Other ML Packages
+
+In our documentation, we state that the mitigation algorithms in Fairlearn are compatible with any package which presents an interface similar to SciKit-Learn.
+Specifically, so long as there's some sort of 'estimator' concept which provides `fit()` and `predict()` methods (the former with a `sample_weight=` argument), then those estimators can be used with our mitigation algorithms.
+However, we do not currently have tests proving that we do offer this sort of compatibility with a variety of popular ML packages.
+
+## Target Packages
+
+We should show compatibility with the following ML packages:
+  - XGBoost
+  - LightGBM
+  - TensorFlow
+  - PyTorch
+
+This is not an exhaustive list, but is a good starting point.
+In all cases, these packages either offer a SciKit-Learn compatible API, or there is a SciKit-Learn compatible wrapper available (such as `skorch` for PyTorch).
+
+## Required Tests
+
+The primary goal of this testing is to show that other ML packages can be used with our mitigation algorithms.
+This does not require in-depth testing, but more a smoke test.
+The tests should verify that, with a reasonable dataset, estimators from each package (possibly only a single example estimator) can be used with our mitigation algorithms, and produce reasonable results.
+A sample test might look a lot like those in `test_exponentiatedgradient_smoke.py` but on a more realistic dataset, and with fewer combinations of constraints and error bounds.
```

Comment:
Why fewer combinations? Might as well be exhaustive...

Reply:
These aren't supposed to be exhaustive. They're supposed to reassure us that we can call Fairlearn using other packages as the underlying estimator. At least in the short term, the goal should be to test more packages, rather than in depth testing of any one package. If and when more specific issues are found, we can certainly add more detailed tests, but that's not the goal right now.